app/views/access_requests/_access_request_api_result_banner.html.erb

```diff
@@ -0,0 +1,31 @@
+<% if @api_result == 'success' %>
+  <div class="govuk-success-summary" aria-labelledby="success-summary-title" role="alert" tabindex="-1" data-module="success-summary">
```

Comment:
Think we need to retain `data-module="error-summary"` to ensure the element is focused on page load from this module - https://github.com/alphagov/govuk-frontend/blob/master/src/components/error-summary/error-summary.js